app/process/watcher/evm/watcher.go

```diff
@@ -346,20 +340,15 @@ func (ew *Watcher) handleBurnLog(eventLog *router.RouterBurn, q qi.Queue) {
 		return
 	}
 
-	var chain *big.Int
-	chain, e := ew.evmClient.ChainID(context.Background())
-	if e != nil {
-		ew.logger.Errorf("[%s] - Failed to retrieve chain ID.", eventLog.Raw.TxHash)
-		return
-	}
+	chain := ew.evmClient.ChainID()
```

Comment:
can we rename this to `nativeChainId`

Reply:
will be fixed in a new PR https://github.com/LimeChain/hashport-validator/pull/412